rsc/src/main/scala/rsc/scalasig/Pickle.scala

```diff
@@ -1123,11 +1123,7 @@ class Pickle private (settings: Settings, mtab: Mtab, sroot1: String, sroot2: St
       )
     }
     def scaseAccessor(sgetterSym: String): s.SymbolInformation = {
-      val saccessorName = {
-        // FIXME: https://github.com/twitter/rsc/issues/99
-        if (settings.abi == Abi211) gensym.caseAccessor(sgetterSym.desc.value)
-        else crash(settings.abi.toString)
-      }
+      val saccessorName = gensym.caseAccessor(sgetterSym.desc.value)
```

Comment:
Issue 99 claims that this isn't fully compatible with 2.12, but the issue is also nearly a year old. Do we need to add any test cases to make sure we are fully implementing 2.12 compatible behavior?

Reply:
No, not for now -- thus I didn't close the issue. Bytecode checks out for util at least.